app/services/doc_auth/error_generator.rb

```diff
@@ -362,12 +400,13 @@ def scan_for_unknown_alerts(response_info)
       unknown_fail_count
     end
 
-    def self.general_error(_liveness_enabled)
-      Errors::GENERAL_ERROR
-    end
+    def attention_with_barcode_result(doc_auth_result, processed_alerts)
```

Comment:
I'm not sure about this method. Is it a direct copy/paste from `def attention_with_barcode?` in `true_id_response.rb`?

Reply:
Pretty much yes! I thought about trying to pass it through to the error generator, but with the complications of:

- needing to also make that change in the mock proofer
- needing to update testing
- knowing that we're going to remove the attention with barcode logic soonish

I didn't think it was worth the lift at this time. If you disagree though I'm happy to take a look at doing so.

Reply:
That logic makes sense to me, could you label it with a comment including this logic. Something like:
`# This method replicates TrueIdResponse::attention_with_barcode? and should be removed/updated when that is.`

Reply:
That's a great idea! Added in 2a4fbc75427d3809ee8debaeda4c8fa8cf78dd38.